fizzbuzz_test.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+from fizzbuzz import fizzbuzz
+
+
+class fizzbuzzTest(unittest.TestCase):
```

Comment:
Name classes in python using Title case and not camel case. Forexample
```py
# good
class FizzBuzTest(unittest.TestCase):
    # class logic

# bad
class fizzbuzzTest(unittest.TestCase):
   # class logic here
```

Reply:
Alright thanks, let me change for #good